README.md

```diff
@@ -58,16 +76,15 @@ See [here](https://docs.ansible.com/ansible/devel/dev_guide/developing_collectio
 
 ### Communication
 
-We have a dedicated Working Group for Ansible development.
+We announce releases and important changes through the [Ansible Bullhorn newsletter](https://github.com/ansible/community/issues/546). Be sure you are subscribed.
+
+Join us in the ``#ansible-community``and other [IRC channels](https://docs.ansible.com/ansible/devel/community/communication.html#irc-channels).
 
-You can find other people interested on the following [Libera.chat](https://libera.chat/) IRC channels -
-- `#ansible` - For general use questions and support.
-- `#ansible-devel` - For discussions on developer topics and code related to features or bugs in ansible-core.
-- `#ansible-community` - For discussions on community topics and community meetings, and for general development questions for community collections.
+We take part in the global quarterly [Ansible Contributor Summit](https://github.com/ansible/community/wiki/Contributor-Summit) virtually or in-person. Track the [Bullhorn newsletter](https://github.com/ansible/community/issues/546) and join us.
```

Comment:
What do you think about actually subscribing to Bullhorn newsletter instead of watching the issue? From top post in issue they can be subscribed here (http://eepurl.com/gZmiEP).

@cybette Is this the official method?

Reply:
Yes, subscribing with [this link](http://eepurl.com/gZmiEP) is the official way. Alternatively you can link to the [News wiki](https://github.com/ansible/community/wiki/News#the-bullhorn). I do not recommend linking to the [issue](https://github.com/ansible/community/issues/546) because as @aminvakil said it can be too noisy for most users, plus we will probably create a new issue soon as the old one is getting quite long.

Reply:
Sounds sensible, changed, thanks!